tests/helpers/open-graph/image-helper-test.php

```diff
@@ -89,4 +87,90 @@ public function test_is_image_url_valid() {
 		);
 	}
 
+	/**
+	 * Tests retrieval of the overridden image size.
+	 *
+	 * @covers ::get_override_image_size
+	 */
+	public function test_get_override_image_size() {
+		Monkey\Functions\expect( 'apply_filters' )
+			->with( 'wpseo_opengraph_image_size', null )
+			->once()
+			->andReturn( 'full' );
+
+		$this->assertEquals( 'full', $this->instance->get_override_image_size() );
+	}
+
+	/**
+	 * Tests retrieval of the overridden image size.
+	 *
+	 * @covers ::get_image_url_by_id
+	 */
+	public function test_get_image_url_by_id_for_invalid_attachment() {
+		$this->image_helper
+			->expects( 'is_valid_attachment' )
+			->with( 1337 )
+			->once()
+			->andReturnFalse();
```

Comment:
In this PR you both use `andReturnFalse()` and `andReturn( false )`. Make consistent?

Reply:
That's because of the `andReturn( false )` is used for a Brain Monkey expectation and that library doesn't have `andReturnFalse()`